source/Xunit.Gherkin.Quick/GherkinException.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Xunit.Gherkin.Quick
+{
+    public class GherkinException : Exception
```

Comment:
I think it's not needed yet. Regardless, classes should be sealed by default preferably. Inheritance requires other design thought process, which I think is not practiced by default.